content/src/Service.Host/views/Index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Pricing</title>
+    <title>template</title>
```

Comment:
Worth doing this with a capital T and then it should get replaced when the template is instantiated